src/api/app/views/webui/shared/bs_requests/_requests_filter.html.haml

```diff
@@ -72,22 +72,24 @@
                                                             label_icon: action_type_icon('submit'),
                                                             checked: selected_filter[:action_type]&.include?('submit')}
 
--# TODO: Temporarily disable list of creators due to performance issues
   .mt-2.mb-2.accordion-item.border-0
     %h6.px-3.py-2.accordion-button.no-style{ data: { 'bs-toggle': 'collapse', 'bs-target': '#request-filter-creator' },
                                                        aria: { expanded: 'true', controls: 'request-filter-creator' } }
       %b Creator
+      .selected-content.small.ms-1
     .px-4.pb-2.accordion-collapse.collapse.show#request-filter-creator
       #request-creator-dropdown
-        = text_field_tag('request-creator-search', nil, autocomplete: 'off', placeholder: 'Search creators...', class: 'form-control mb-2')
+        = render partial: 'webui/shared/autocomplete', locals: { html_id: 'creators_search', label: 'Creators:',
```

Comment:
Would be nice to make the `label` parameter optional, so we can omit in this case instead. Otherwise it looks like the following

![image](https://github.com/user-attachments/assets/40ce3a00-8285-4c6c-9294-726e845e0110)

where the `autocomplete` label is the `Creators:` small one, and we can't omit the other one instead.

Reply:
Alternatively, to make it simpler (a bit redundant though)
```suggestion
        = render partial: 'webui/shared/autocomplete', locals: { html_id: 'creators_search', label: 'Username:',
```

or `Search by username`

Reply:
@ncounter I simply removed the creator label. Please have another look :)

Reply:
Cool! :+1: